assets/src/apps/v2/triptych.tsx

```diff
@@ -7,33 +7,51 @@ import ReactDOM from "react-dom";
 import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 
 import { usePlayerName } from "Hooks/outfront";
+import useIsOnScreen from "Hooks/v2/use_is_on_screen";
 import { isTriptych } from "Util/outfront";
 
 import { MappingContext } from "Components/v2/widget";
 import {
   ResponseMapper,
   ResponseMapperContext,
 } from "Components/v2/screen_container";
+import { imagePath } from "Util/util";
 
 import ScreenPage from "Components/v2/screen_page";
 import MultiScreenPage from "Components/v2/multi_screen_page";
 import SimulationScreenPage from "Components/v2/simulation_screen_page";
 import Viewport from "Components/v2/triptych/viewport";
 
 import FullScreen from "Components/v2/basic_layouts/full_screen";
+import TriptychThreePane from "Components/v2/triptych/triptych_three_pane";
 
 import PageLoadNoData from "Components/v2/triptych/page_load_no_data";
 import NoData from "Components/v2/triptych/no_data";
 
 import Placeholder from "Components/v2/placeholder";
 import TrainCrowding from "Components/v2/train_crowding";
+import EvergreenContent from "Components/v2/evergreen_content";
+
+const customizeEvergreenProps = (WrappedComponent: React.ElementType) => {
+  return (props: { asset_url: string }) => {
+    const modifiedUrl = props.asset_url.replace("assets/static/images/", "");
+    const dupReadyUrl = imagePath(modifiedUrl);
+
+    const isPlaying = useIsOnScreen();
+    return <WrappedComponent asset_url={dupReadyUrl} isPlaying={isPlaying} />;
+  };
+};
```

Comment:
I suppose. Since this HOC is so simple and shouldn't be reused, I thought it fine and readable to keep it internal to the Triptych file. But I see the generalization is unneeded.